src/modules/filters/components/FilterBar/FilterBar.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import useFelaEnhanced from 'hooks/useFelaEnhanced';
+import { Select, Form, Space } from 'antd';
+
+import uuid from 'react-uuid';
```

Comment:
is it still uuid here? shouldnt it be switched for nano?

Reply:
Removed